modules/standard/Regexp.chpl

```diff
@@ -387,7 +387,7 @@ extern const QIO_REGEXP_ANCHOR_BOTH:c_int;
 pragma "no doc"
 extern record qio_regexp_string_piece_t {
   var offset:int(64); // counting from 0, -1 means "NULL"
-  var len:int(64);
+  var buffLen:int(64);
```

Comment:
let's not change this here

Reply:
I have done the changes requested. I also ran tests ```start_test test/types/string/kbrady``` ```start_test test/release/examples/primers``` and ```start_test test/types/bytes``` successfully in my system. Kindly review and notify me of any more changes required.

Reply:
Also, I accidently deleted the ```rst/``` a couple of days back, hence, the large changes that are shown [here](https://github.com/chapel-lang/chapel/pull/15225/commits/b289ca64cc608cae0b3a62b7924bc96c051f35ee). In the mentioned commit, only ```Regexp.chpl``` file has been modified. Kindly ignore the others. Thank you